src/ir/possible-contents.h

```diff
@@ -608,6 +615,10 @@ template<> struct hash<wasm::TagLocation> {
   }
 };
 
+template<> struct hash<wasm::CaughtExnRefLocation> {
+  size_t operator()(const wasm::CaughtExnRefLocation& loc) const { return 0; }
```

Comment:
0 is actually risky, as if we had other such things then they'd also use 0 in the same natural way, and all collide. How about `std::hash<const char*>()("caught-exnref-location")` - ? That hashes the pointer, and that pointer should be unique as other things wouldn't use that text presumably.